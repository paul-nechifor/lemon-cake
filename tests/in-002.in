(= d (dict 10 20 20 40 30 60))
(in 20 d)
(in 30 d)
(in 10 d)
(in 40 d)
