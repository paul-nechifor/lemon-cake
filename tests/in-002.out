(dict 10 20 20 40 30 60)
1
1
1
0
